src/ember-app.js

```diff
@@ -450,6 +450,8 @@ function createShoebox(doc, fastbootInfo) {
 
     let value = shoebox[key];
     let textValue = JSON.stringify(value);
+    if (textValue === undefined) { continue; }
```

Comment:
> That's the only reason why I proposed for logging it since app is expecting it in the shoebox.

I'm curious why we wouldn't error? (if it is expected)